src/main/java/client/Character.java

```diff
@@ -323,6 +325,45 @@ public void onAnnounceStatPoolUpdate() {
         setPosition(new Point(0, 0));
     }
 
+    static {
+        if (YamlConfig.config.server.USE_EXP_GAIN_LOG) {
+            SaveExpLogToDBThread expThread = new SaveExpLogToDBThread();
+            expThread.setPriority(Thread.MIN_PRIORITY); // Set to the lowest priority
+            expThread.start();
```

Comment:
I'd like to avoid adding stuff to `Character.java`, especially since this is an optional feature. Could keep it in some manager-type class and only expose a method to add an explog which would be called from `Character.java`.
I'm not enforcing how you should structure, I'm just saying that's what I would have done.